menu.js

```diff
@@ -39,4 +33,143 @@ function chameleon() {
+    LongDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    LongDescriptionDesk: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.<br><br>Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.',
+    Image: './images/card4desk.png',
+    Features: './images/canopydesk4.png',
+    Technologies: ['./images/htmltag.png', './images/csstag.png', './images/javascripttag.png', './images/rubytag.png'],
+    SeeProject: 'See project',
+    Live: 'https://www.google.com',
+    Source: 'https://github.com',
+    CloseButton: './images/closePopUpIcon.png',
+  },
+];
+
+// Cards Template
+
+function generatePopUpTags(technologies) {
+  return technologies
+    .map((technology, index) => `<li><img class="programs pro${index}" src="${technology}"></li>`)
+    .join('');
+}
+
+function generatePopUpCard(card) {
+  const popUpCard = document.createElement('div');
+  popUpCard.classList.add('popUpCard');
+
+  popUpCard.innerHTML = `<div class="popUpContent">
+     <div class="topPopUp">
+       <h2 class="popEichTwo">${card.Name}</h2>
+       <img class="popUpClose" src="${card.CloseButton}" alt="">
+     </div>
+    <div><img class="canopy" src="${card.Features}" alt=""></div>
+    <div class="orange">
+      <img class="mobileImage" src="${card.Image}" alt="">
+      <img class="deskImage" src="${card.Image}" alt="">
+    </div>
+    <div class="popFooterDesk">
+    <p class="popParMobile">${card.LongDescription}</p>
+    <p class="popParDesk">${card.LongDescriptionDesk}</p>
+    <div class="popFooterDeskRight">
+    <ul class="popUpTags">
+        ${generatePopUpTags(card.Technologies)}
+    </ul>
+    <img class="popSeparator" src="./images/sepPopUpFooter.png" alt="">
+       <div class="footerButtonsContainer">
+        <a class="popUpLink" href="${card.Live}" target="_blank"><button class="footerButtons">See Live</button></a>
+        <a class="popUpLink" href="${card.Source}" target="_blank"><button class="footerButtons">See Source</button></a>
+      </div>
+      </div>
```

Comment:
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] You have done great so far. :+1: Your pop-up works great, but there are a few changes you need to make before moving to the next milestone.

- As stated in the project requirement, you need to replicate the design as much as you can, currently, the `see live` and `See Source` buttons don't have the icons as shown in the Figma design, I'll suggest implementing that. 
- Also the button hight is small compared to that of the Figma design, I'll suggest looking into that too

 - - YOUR CURRENT DESIGN:
 
![Screenshot from 2023-06-24 18-22-05](https://github.com/AdrianScharff/My-portfolio-Adrian-Juarez/assets/107632780/ddd663af-24b5-4aa6-96f4-e380e7771264)

 - - FIGMA DESKTOP:
![Screenshot from 2023-06-24 18-22-17](https://github.com/AdrianScharff/My-portfolio-Adrian-Juarez/assets/107632780/7205cda8-2aaa-4576-bfb5-9e7df39c214a)

 - - FIGMA MOBILE:
![Selection_075](https://github.com/AdrianScharff/My-portfolio-Adrian-Juarez/assets/107632780/938c61b3-d7f3-457c-9ba2-475ba87977ab)


- [ ] Also the buttons are missing the button interaction attributes like `hover`, I'll suggest you implement that too. :+1: :+1: